tokio/src/runtime/time/tests/mod.rs

```diff
@@ -202,6 +202,27 @@ fn reset_future() {
     })
 }
 
+#[test]
+fn reset_timer_and_drop() {
+    model(|| {
```

Comment:
You should only use loom for tests that spawn threads.

Reply:
The `#[cfg(not(loom))]` is been added for the test `reset_timer_and_drop`.

Reply:
You can get rid of the `model` call as well.

Reply:
Done. Thanks.